dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/jobService/GetJob.kt

```diff
@@ -68,7 +68,7 @@ class GetJob(private val getJobData: JobData) : JobService(), PrepStage<JobData>
     }
 
     override fun transfer(job: Ds3ClientHelpers.Job) {
-        title.set("Transferring GET Job ${job.jobId}")
+        title.set("Transferring GET Job ${job.jobId} started at ${getJobData.dateTimeUtils().nowAsString()}")
```

Comment:
Should we be getting this from a resource string?

Reply:
I didn't think that through enough, patching so it's consistent with the log.